Agent GET endpoints AIT fails on versions higher than v4.13.0

## Description

After the changes introduced at https://github.com/wazuh/wazuh/pull/29166, a new API IT test case was added for the endpoints related to the `GET` method used to get agent information. This test case contains a hardcoded value for the latest Wazuh version (`v4.13.0`) that breaks when running the test in a higher branch version. Example in `4.13.1`:

<details><summary>test_agent_GET_endpoints</summary>

```console
(4.13.1)$ pytest test_agent_GET_endpoints.tavern.yaml 
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.10.12, pytest-7.3.1, pluggy-1.5.0
rootdir: /home/fdalmau/Git/wazuh/api/test/integration
configfile: pytest.ini
plugins: html-2.1.1, asyncio-0.18.1, tavern-1.23.5, cov-4.1.0, metadata-3.1.1, anyio-4.1.0, trio-0.8.0
asyncio: mode=auto
collected 95 items                                                                                                                                                                                                

test_agent_GET_endpoints.tavern.yaml FF..........................................F..................................................                                                                        [100%]

==================================================================================================== FAILURES =====================================================================================================
_________________________________________________________ /home/fdalmau/Git/wazuh/api/test/integration/test_agent_GET_endpoints.tavern.yaml::GET /agents __________________________________________________________
Format variables:

Source test stage (line 62):
  - name: Get all agents sorted by version in descending order
    request:
      verify: False
      <<: *get_agents
      params:
        sort: -version,id
        select: version,id
    response:
      status_code: 200
      json:
        error: 0
        data:
          affected_items:
            - id: '004'
              version: 'Wazuh v4.13.0'
            - id: '003'
              version: 'Wazuh v4.13.0'
            - id: '002'
              version: 'Wazuh v4.13.0'
            - id: '001'
              version: 'Wazuh v4.13.0'
            - id: '000'
              version: 'Wazuh v4.13.0'
            - id: '008'
              version: 'Wazuh v3.13.2'
            - id: '007'
              version: 'Wazuh v3.13.2'
            - id: '006'
              version: 'Wazuh v3.13.2'
            - id: '005'
              version: 'Wazuh v3.13.2'
            - id: '010'
              version: 'Wazuh v3.9.2'
            - id: '009'
              version: 'Wazuh v3.9.2'
            - id: '012'
            - id: '011'
          failed_items: []
          total_affected_items: 13
          total_failed_items: 0


Formatted stage:
  name: Get all agents sorted by version in descending order
  request:
    headers:
      Authorization: Bearer eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNzUwMTgyNjI1LCJleHAiOjE3NTAxODM1MjUsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbMV0sInJiYWNfbW9kZSI6IndoaXRlIn0.AGYc-_4p-vYji6owOzdgTEWjarANW3XdiRlEhu5gCFINrU1uhHtovdjiWS_8aRdX8eEv3dhkN85PQ_BX95cjw-FAAIBih9ZeK7gbC2gFolM6Uo9ybHMOMB1uD3ZDxDOPb8jddb0GhTU0h0wLZI7i6ZoYcgc0r_9Y8ZMoahbwPpSyw0m6
    method: GET
    params:
      select: version,id
      sort: -version,id
    url: 'https://localhost:55000/agents'
    verify: false
  response:
    json:
      data:
        affected_items:
        - id: '004'
          version: Wazuh v4.13.0
        - id: '003'
          version: Wazuh v4.13.0
        - id: '002'
          version: Wazuh v4.13.0
        - id: '001'
          version: Wazuh v4.13.0
        - id: '000'
          version: Wazuh v4.13.0
        - id: 008
          version: Wazuh v3.13.2
        - id: '007'
          version: Wazuh v3.13.2
        - id: '006'
          version: Wazuh v3.13.2
        - id: '005'
          version: Wazuh v3.13.2
        - id: '010'
          version: Wazuh v3.9.2
        - id: 009
          version: Wazuh v3.9.2
        - id: '012'
        - id: '011'
        failed_items: []
        total_affected_items: 13
        total_failed_items: 0
      error: 0
    status_code: 200

Errors:
E   tavern.util.exceptions.TestFailError: Test 'Get all agents sorted by version in descending order' failed:
    - List item(s) not present in response: [{'id': '004', 'version': 'Wazuh v4.13.0'}, {'id': '003', 'version': 'Wazuh v4.13.0'}, {'id': '002', 'version': 'Wazuh v4.13.0'}, {'id': '001', 'version': 'Wazuh v4.13.0'}, {'id': '000', 'version': 'Wazuh v4.13.0'}, {'id': '008', 'version': 'Wazuh v3.13.2'}, {'id': '007', 'version': 'Wazuh v3.13.2'}, {'id': '006', 'version': 'Wazuh v3.13.2'}, {'id': '005', 'version': 'Wazuh v3.13.2'}, {'id': '010', 'version': 'Wazuh v3.9.2'}, {'id': '009', 'version': 'Wazuh v3.9.2'}, {'id': '012'}, {'id': '011'}]
------------------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------------------
ERROR    tavern.response.base:base.py:41 List item(s) not present in response: [{'id': '004', 'version': 'Wazuh v4.13.0'}, {'id': '003', 'version': 'Wazuh v4.13.0'}, {'id': '002', 'version': 'Wazuh v4.13.0'}, {'id': '001', 'version': 'Wazuh v4.13.0'}, {'id': '000', 'version': 'Wazuh v4.13.0'}, {'id': '008', 'version': 'Wazuh v3.13.2'}, {'id': '007', 'version': 'Wazuh v3.13.2'}, {'id': '006', 'version': 'Wazuh v3.13.2'}, {'id': '005', 'version': 'Wazuh v3.13.2'}, {'id': '010', 'version': 'Wazuh v3.9.2'}, {'id': '009', 'version': 'Wazuh v3.9.2'}, {'id': '012'}, {'id': '011'}]
Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert {'data': {'af...was returned'} == {'data': {'af...}, 'error': 0}
  Omitting 1 identical items, use -vv to show
  Differing items:
  {'data': {'affected_items': [{'id': '004', 'version': 'Wazuh v4.13.1'}, {'id': '003', 'version': 'Wazuh v4.13.1'}, {'i...d': '008', 'version': 'Wazuh v3.13.2'}, ...], 'failed_items': [], 'total_affected_items': 13, 'total_failed_items': 0}} != {'data': {'affected_items': [{'id': '004', 'version': 'Wazuh v4.13.0'}, {'id': '003', 'version': 'Wazuh v4.13.0'}, {'i...d': '008', 'version': 'Wazuh v3.13.2'}, ...], 'failed_items': [], 'total_affected_items': 13, 'total_failed_items': 0}}
  Left contains 1 more item:
  {'messa...
  
  ...Full output truncated (2 lines hidden), use '-vv' to show

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert {'affected_it...led_items': 0} == {'affected_it...led_items': 0}
  Omitting 3 identical items, use -vv to show
  Differing items:
  {'affected_items': [{'id': '004', 'version': 'Wazuh v4.13.1'}, {'id': '003', 'version': 'Wazuh v4.13.1'}, {'id': '002'...version': 'Wazuh v4.13.1'}, {'id': '000', 'version': 'Wazuh v4.13.1'}, {'id': '008', 'version': 'Wazuh v3.13.2'}, ...]} != {'affected_items': [{'id': '004', 'version': 'Wazuh v4.13.0'}, {'id': '003', 'version': 'Wazuh v4.13.0'}, {'id': '002'...version': 'Wazuh v4.13.0'}, {'id': '000', 'version': 'Wazuh v4.13.0'}, {'id': '008', 'version': 'Wazuh v3.13.2'}, ...]}
  Use -v to get more diff

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert [{'id': '004'...3.13.2'}, ...] == [{'id': '004'...3.13.2'}, ...]
  At index 0 diff: {'version': 'Wazuh v4.13.1', 'id': '004'} != {'id': '004', 'version': 'Wazuh v4.13.0'}
  Use -v to get more diff

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/response/base.py", line 97, in recurse_check_key_match
    check_keys_match_recursive(expected_block, block, [], strict)
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 464, in check_keys_match_recursive
    check_keys_match_recursive(
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 464, in check_keys_match_recursive
    check_keys_match_recursive(
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 516, in check_keys_match_recursive
    raise exceptions.KeyMismatchError(msg) from e
tavern.util.exceptions.KeyMismatchError: List item(s) not present in response: [{'id': '004', 'version': 'Wazuh v4.13.0'}, {'id': '003', 'version': 'Wazuh v4.13.0'}, {'id': '002', 'version': 'Wazuh v4.13.0'}, {'id': '001', 'version': 'Wazuh v4.13.0'}, {'id': '000', 'version': 'Wazuh v4.13.0'}, {'id': '008', 'version': 'Wazuh v3.13.2'}, {'id': '007', 'version': 'Wazuh v3.13.2'}, {'id': '006', 'version': 'Wazuh v3.13.2'}, {'id': '005', 'version': 'Wazuh v3.13.2'}, {'id': '010', 'version': 'Wazuh v3.9.2'}, {'id': '009', 'version': 'Wazuh v3.9.2'}, {'id': '012'}, {'id': '011'}]
_______________________________________________ /home/fdalmau/Git/wazuh/api/test/integration/test_agent_GET_endpoints.tavern.yaml::GET /agents with single agent id _______________________________________________
Format variables:

Source test stage (line 1381):
  - name: Assert active and never_connected agents do not have disconnection_time
    request:
      verify: False
      <<: *get_agent
      params:
        select: "disconnection_time"
        status: "active,never_connected"
    response:
      verify_response_with:

        function: tavern_utils:test_validate_key_not_in_response
        extra_kwargs:
          key: 'disconnection_time'
      status_code: 200



Formatted stage:
  name: Assert active and never_connected agents do not have disconnection_time
  request:
    headers:
      Authorization: Bearer eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNzUwMTgyNjI1LCJleHAiOjE3NTAxODM1MjUsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbMV0sInJiYWNfbW9kZSI6IndoaXRlIn0.AOQ1AgFxyASYfh-Q76YJT811FgOb4j9FVEnzBqmnETThPjPahj9-1V4n67K8Uaf6ZNhJepJ6ndY13BOYPKFqEkn5AEgoTxTYYoPERVaVqy6tyWxT9v5ipHl1hfOoWIVC1tNV4XKkiJ2uCUW3knHNSzas4fk2n6Na9wHQTvEZWHnCFtmC
    method: GET
    params:
      select: disconnection_time
      status: active,never_connected
    url: 'https://localhost:55000/agents'
    verify: false
  response:
    status_code: 200
    verify_response_with:
      extra_kwargs:
        key: disconnection_time
      function: tavern_utils:test_validate_key_not_in_response

Errors:
E   tavern.util.exceptions.TestFailError: Test 'Assert active and never_connected agents do not have disconnection_time' failed:
    - Error calling validate function '<function test_validate_key_not_in_response at 0x735635aacca0>':
        Traceback (most recent call last):
          File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/response/base.py", line 152, in _maybe_run_validate_functions
            vf(response)
          File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/extfunctions.py", line 98, in inner
            result = func(response, *args, **kwargs)
          File "/home/fdalmau/Git/wazuh/api/test/integration/tavern_utils.py", line 395, in test_validate_key_not_in_response
            assert all(key not in item for item in response.json()["data"]["affected_items"])
        AssertionError
------------------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------------------
ERROR    tavern.response.base:base.py:41 Error calling validate function '<function test_validate_key_not_in_response at 0x735635aacca0>':
    Traceback (most recent call last):
      File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/response/base.py", line 152, in _maybe_run_validate_functions
        vf(response)
      File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/extfunctions.py", line 98, in inner
        result = func(response, *args, **kwargs)
      File "/home/fdalmau/Git/wazuh/api/test/integration/tavern_utils.py", line 395, in test_validate_key_not_in_response
        assert all(key not in item for item in response.json()["data"]["affected_items"])
    AssertionError
Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/response/base.py", line 152, in _maybe_run_validate_functions
    vf(response)
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/extfunctions.py", line 98, in inner
    result = func(response, *args, **kwargs)
  File "/home/fdalmau/Git/wazuh/api/test/integration/tavern_utils.py", line 395, in test_validate_key_not_in_response
    assert all(key not in item for item in response.json()["data"]["affected_items"])
AssertionError
_____________________________________ /home/fdalmau/Git/wazuh/api/test/integration/test_agent_GET_endpoints.tavern.yaml::GET /groups/{group_id}/agents {sort,select} version ______________________________________
Format variables:

Source test stage (line 3404):
  - name: Filter groups, sort
    request:
      verify: False
      <<: *groups_id_request
      params:
        sort: version
    response:
      status_code: 200
      json:
        error: 0
        data:
          affected_items:
            - version: 'Wazuh v3.13.2'
            - version: 'Wazuh v3.13.2'
            - version: 'Wazuh v3.13.2'
            - version: 'Wazuh v4.13.0'
          failed_items: []
          total_affected_items: 4
          total_failed_items: 0



Formatted stage:
  name: Filter groups, sort
  request:
    headers:
      Authorization: Bearer eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNzUwMTgyNjM0LCJleHAiOjE3NTAxODM1MzQsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbMV0sInJiYWNfbW9kZSI6IndoaXRlIn0.AFdX1mzOSpEvBCArNTxvf6Q-JlzvBMp5fdBuy69CFdKU7H5UD9AixgxezNgsgCj0sZc7McphdbTQZk0sWM50YkG6AZ8eEq-H74alHkn_qXCMsIMPsn2lqno2RUXcqCL_WPKOOJvAImRIXEomdZgUB81snZHgqgvO94LGm2Js8J4a_UNg
    method: GET
    params:
      sort: version
    url: 'https://localhost:55000/groups/group3/agents'
    verify: false
  response:
    json:
      data:
        affected_items:
        - version: Wazuh v3.13.2
        - version: Wazuh v3.13.2
        - version: Wazuh v3.13.2
        - version: Wazuh v4.13.0
        failed_items: []
        total_affected_items: 4
        total_failed_items: 0
      error: 0
    status_code: 200

Errors:
E   tavern.util.exceptions.TestFailError: Test 'Filter groups, sort' failed:
    - List item(s) not present in response: [{'version': 'Wazuh v4.13.0'}]
------------------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------------------
ERROR    tavern.response.base:base.py:41 List item(s) not present in response: [{'version': 'Wazuh v4.13.0'}]
Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert {'data': {'af...was returned'} == {'data': {'af...}, 'error': 0}
  Omitting 1 identical items, use -vv to show
  Differing items:
  {'data': {'affected_items': [{'configSum': '26ed62a3b5242d5d17e2dbe5b353e81c', 'dateAdd': '1970-01-01T00:00:00+00:00',...00:00', 'group': ['default', 'group3'], ...}], 'failed_items': [], 'total_affected_items': 4, 'total_failed_items': 0}} != {'data': {'affected_items': [{'version': 'Wazuh v3.13.2'}, {'version': 'Wazuh v3.13.2'}, {'version': 'Wazuh v3.13.2'}, {'version': 'Wazuh v4.13.0'}], 'failed_items': [], 'total_affected_items': 4, 'total_failed_items': 0}}
  Left contains 1 more item:
  {'message': 'All selected agents information was returned'}
  Use -v to get more diff

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert {'affected_it...led_items': 0} == {'affected_it...led_items': 0}
  Omitting 3 identical items, use -vv to show
  Differing items:
  {'affected_items': [{'configSum': '26ed62a3b5242d5d17e2dbe5b353e81c', 'dateAdd': '1970-01-01T00:00:00+00:00', 'group':... '1970-01-01T00:00:00+00:00', 'disconnection_time': '2025-06-17T17:48:42+00:00', 'group': ['default', 'group3'], ...}]} != {'affected_items': [{'version': 'Wazuh v3.13.2'}, {'version': 'Wazuh v3.13.2'}, {'version': 'Wazuh v3.13.2'}, {'version': 'Wazuh v4.13.0'}]}
  Use -v to get more diff

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert [{'configSum'...roup3'], ...}] == [{'version': ...zuh v4.13.0'}]
  At index 0 diff: {'os': {'arch': 'x86_64', 'build': '', 'codename': 'Bionic Beaver', 'major': '18', 'minor': '04', 'name': 'Ubuntu', 'platform': 'ubuntu', 'uname': 'Linux |wazuh-agent6 |6.12.10-76061203-generic |#202412060638~1748542656~22.04~663e4dc SMP PREEMPT_DYNAMIC Thu M |x86_64', 'version': '18.04.5 LTS'}, 'group_config_status': 'synced', 'node_name': 'worker2', 'version': 'Wazuh v3.13.2', 'name': 'wazuh-agent6', 'group': ['default', 'group2', 'group3'], 'status': 'active', 'mergedSum': 'ef9282f84836b37c10b3caab62c699e2', 'manager': 'wazuh-worker2', 'ip': '172.18....
  
  ...Full output truncated (2 lines hidden), use '-vv' to show

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/response/base.py", line 97, in recurse_check_key_match
    check_keys_match_recursive(expected_block, block, [], strict)
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 464, in check_keys_match_recursive
    check_keys_match_recursive(
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 464, in check_keys_match_recursive
    check_keys_match_recursive(
  File "/home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/tavern/util/dict_util.py", line 516, in check_keys_match_recursive
    raise exceptions.KeyMismatchError(msg) from e
tavern.util.exceptions.KeyMismatchError: List item(s) not present in response: [{'version': 'Wazuh v4.13.0'}]
================================================================================================ warnings summary =================================================================================================
../../../../../venv/unittest-env-3.10/lib/python3.10/site-packages/_pytest/nodes.py:642
  /home/fdalmau/venv/unittest-env-3.10/lib/python3.10/site-packages/_pytest/nodes.py:642: PytestRemovedIn8Warning: The (fspath: py.path.local) argument to YamlFile is deprecated. Please use the (path: pathlib.Path) argument instead.
  See https://docs.pytest.org/en/latest/deprecations.html#fspath-argument-for-node-constructors-replaced-with-pathlib-path
    return super().from_parent(parent=parent, fspath=fspath, path=path, **kw)

test_agent_GET_endpoints.tavern.yaml: 95 warnings
  <frozen importlib._bootstrap>:283: DeprecationWarning: the load_module() method is deprecated and slated for removal in Python 3.12; use exec_module() instead

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================================================================================= short test summary info =============================================================================================
FAILED test_agent_GET_endpoints.tavern.yaml::GET /agents
FAILED test_agent_GET_endpoints.tavern.yaml::GET /agents with single agent id
FAILED test_agent_GET_endpoints.tavern.yaml::GET /groups/{group_id}/agents {sort,select} version
============================================================================== 3 failed, 92 passed, 96 warnings in 953.23s (0:15:53) ==============================================================================
```

</details>

This issue aims to fix this by parametrizing the latest Wazuh version used to launch the tests.


## Checks
<!-- Do not modify, this will be ticked during development -->
The following elements have been updated or reviewed (should also be checked if no modification is required):
- [x] Tests (unit tests, API integration tests).
- [x] Changelog.
- [x] Documentation.
- [x] Integration test mapping (using `api/test/integration/mapping/_test_mapping.py`).

--- a/api/test/integration/test_agent_GET_endpoints.tavern.yaml
+++ b/api/test/integration/test_agent_GET_endpoints.tavern.yaml
@@ -3,6 +3,19 @@ test_name: GET /agents
 
 stages:
 
+  - name: Get wazuh version
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/"
+      method: GET
+      headers:
+        Authorization: "Bearer {test_login_token}"
+    response:
+      status_code: 200
+      save:
+        json:
+          wazuh_version: data.api_version
+
   - name: Get all agents
     request: &get_agents
       verify: False
@@ -74,15 +87,15 @@ stages:
         data:
           affected_items:
             - id: '004'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '003'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '002'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '001'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '000'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '008'
               version: 'Wazuh v3.13.2'
             - id: '007'
@@ -129,15 +142,15 @@ stages:
             - id: '008'
               version: 'Wazuh v3.13.2'
             - id: '000'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '001'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '002'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '003'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
             - id: '004'
-              version: 'Wazuh v4.13.0'
+              version: 'Wazuh v{wazuh_version:s}'
           failed_items: [ ]
           total_affected_items: 13
           total_failed_items: 0
@@ -3402,6 +3415,19 @@ test_name: GET /groups/{group_id}/agents {sort,select} version
 
 stages:
 
+  - name: Get wazuh version
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/"
+      method: GET
+      headers:
+        Authorization: "Bearer {test_login_token}"
+    response:
+      status_code: 200
+      save:
+        json:
+          wazuh_version: data.api_version
+
   - name: Filter groups, sort
     request:
       verify: False
@@ -3417,7 +3443,7 @@ stages:
             - version: 'Wazuh v3.13.2'
             - version: 'Wazuh v3.13.2'
             - version: 'Wazuh v3.13.2'
-            - version: 'Wazuh v4.13.0'
+            - version: 'Wazuh v{wazuh_version:s}'
           failed_items: []
           total_affected_items: 4
           total_failed_items: 0

